src/implementations/deploy/token-registry/token-registry.test.ts

```diff
@@ -1,10 +1,7 @@
-import { deployTokenRegistry } from "./token-registry";
-import { join } from "path";
-import { TradeTrustErc721Factory } from "@govtechsg/token-registry";
-import { Wallet } from "ethers";
+import { deployTokenRegistry, encodeInitParams } from "./token-registry";
+import { Contract } from "ethers";
 import { DeployTokenRegistryCommand } from "../../../commands/deploy/deploy.types";
-
-jest.mock("@govtechsg/token-registry");
+import { DeploymentEvent } from "@govtechsg/token-registry/dist/contracts/contracts/utils/TDocDeployer";
 
 const deployParams: DeployTokenRegistryCommand = {
   registryName: "Test",
```

Comment:
let's update all param mentions of `ropsten` to `goerli`? also, should `delete process.env.OA_PRIVATE_KEY;` be in `beforeEach`?